grpc-kotlin/src/test/proto/com/linecorp/armeria/grpc/testing/messages.proto

```diff
@@ -0,0 +1,282 @@
+}
+
+// Server-streaming request.
+message StreamingOutputCallRequest {
+  // Desired payload type in the response from the server.
+  // If response_type is RANDOM, the payload from each response in the stream
+  // might be of different types. This is to simulate a mixed type of payload
+  // stream.
+  PayloadType response_type = 1;
+
+  // Configuration for each expected response message.
+  repeated ResponseParameters response_parameters = 2;
+
+  // Optional input payload sent along with the request.
+  Payload payload = 3;
+
+  // Compression algorithm to be used by the server for the response (stream)
+  CompressionType response_compression = 6;
+
+  // Whether server should return a given status
+  EchoStatus response_status = 7;
+
+  // Arbitrary options for this request.
+  map<string, int32> options = 8;
+}
+
+// Server-streaming response, as configured by the request and parameters.
+message StreamingOutputCallResponse {
+  // Payload to increase response size.
+  Payload payload = 1;
+}
+
+// For reconnect interop test only.
+// Client tells server what reconnection parameters it used.
+message ReconnectParams {
+  int32 max_reconnect_backoff_ms = 1;
+}
+
+// For reconnect interop test only.
+// Server tells client whether its reconnects are following the spec and the
+// reconnect backoffs it saw.
+message ReconnectInfo {
+  bool passed = 1;
+  repeated int32 backoff_ms = 2;
+}
+
+enum TestEnum {
+  ZERO = 0;
+  ONE = 1;
+  TWO = 2;
+}
+
+message TestMessage {
+  message Nested {
+    string string = 1;
+  }
+
+  bool bool = 1;
+  int32 int32 = 2;
+  int64 int64 = 3;
+  uint32 uint32 = 4;
+  uint64 uint64 = 5;
+  sint32 sint32 = 6;
+  sint64 sint64 = 7;
+  fixed32 fixed32 = 8;
+  fixed64 fixed64 = 9;
+  float float = 10;
+  double double = 11;
+  string string = 12;
+  bytes bytes = 13;
+  TestEnum test_enum = 14;
+  Nested nested = 15;
+  repeated string strings = 16;
+  map<string, int32> map = 17;
+  TestMessage self = 18;
+}
+
+message ExtendedTestMessage {
+  message Nested {
+    string string = 1;
+  }
+
+  message NestedSelf {
+    ExtendedTestMessage self = 1;
+  }
+
+  message NestedNestedSelf {
+    NestedSelf nested_self = 1;
+  }
+
+  bool bool = 1;
+  int32 int32 = 2;
+  int64 int64 = 3;
+  uint32 uint32 = 4;
+  uint64 uint64 = 5;
+  sint32 sint32 = 6;
+  sint64 sint64 = 7;
+  fixed32 fixed32 = 8;
+  fixed64 fixed64 = 9;
+  float float = 10;
+  double double = 11;
+  string string = 12;
+  bytes bytes = 13;
+  TestEnum test_enum = 14;
+  Nested nested = 15;
+  TestMessage complex_other_message = 16;
+  repeated string strings = 17;
+  repeated Nested nesteds = 18;
+  repeated ExtendedTestMessage selves = 19;
+  map<string, int32> string_to_int_map = 20;
+  map<int32, string> int_to_string_map = 21;
+  map<string, Nested> message_map = 22;
+  map<string, ExtendedTestMessage> self_map = 23;
+  ExtendedTestMessage self = 24;
+  NestedSelf nested_self = 25;
+  NestedNestedSelf nested_nested_self = 26;
+}
```

Comment:
```suggestion
```

Reply:
Done in [c5aa9ab](https://github.com/line/armeria/pull/4724/commits/c5aa9ab3a1e5d1d4bfa41274d85c350ec4da290d)!